docs/plugins/npe2_getting_started.md

```diff
@@ -0,0 +1,286 @@
+(npe2-getting-started)=
+
+# npe2 getting started guide
+
+This guide will walk you through the steps to write an `npe2`-style napari
+plugin from scratch.
+
+At the end of this guide, you will have a working plugin that can be installed
+with `pip` and autodetected by napari.
+
+## Overview
+
+Plugins are just python packages. They include contributions that napari
+may use when performing tasks (like reading data), and a "manifest" file that
+tells napari where in the package to find these contributions.
+
+This guide covers `npe2`-style plugins. `npe2` plugins declare the
+functionality that they contribute in a file called the plugin manifest.
+
+Creating a new plugin involves the following steps:
+
+1. Configure a python package to use an npe2 manifest (e.g. by editing
+   `setup.cfg`):
+   - Create the _entry point group_.
+   - Make sure the manifest file is added to `package_data`.
+2. Create the plugin manifest file.
+
+This guide will walk you through the steps using a [cookiecutter][] template.
```

Comment:
I'm sure you know this but, link here.

Reply:
```suggestion
This guide will walk you through the steps using a [cookiecutter] template.
```

Reply:
It turns out the two link syntaxes are equivalent, but is seems the standard we use is `[cookiecutter]`.  So I switched to that.  The url is listed at the bottom of the markdown file.